src/envo/envo-edit.owl

```diff
@@ -27848,13 +27869,157 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000000> <http://purl.obolibrar
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000003> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000003> "https://en.wikipedia.org/wiki/Nivation")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000003> "nivation"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000003> <http://purl.obolibrary.org/obo/ENVO_02500000>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000004> (glacial ice)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000004> "Any water ice that is part of a glacier.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000004> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000004> "https://en.wikipedia.org/wiki/Glacier")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000004> "glacial ice"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000004> <http://purl.obolibrary.org/obo/ENVO_01000293>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000005> (erosionally enriched glacial ice)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000005> "Any glacial ice which contains material liberated during a glacial erosion process.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000005> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000005> "https://www.asf.alaska.edu/blog/why-is-glacier-ice-blue/")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000005> "erosionally enriched glacial ice"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000005> <http://purl.obolibrary.org/obo/ENVO_03000004>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000006> (glacial erosion process)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000006> "An erosion process during which the movement of a glacier across a terrestrial surface causes the removal of material from that surface.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000006> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000006> "glacial erosion process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000006> <http://purl.obolibrary.org/obo/ENVO_02500028>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000007> (blue ice)
```

Comment:
Add a comment stating that the 'appearance' is to be understood as that within the visual range of humans